enrollment-server-onboarding/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -8,7 +8,7 @@
 
 		<resources>
 			<!-- use WAR provided Bouncy Castle -->
-			<resource-root path="WEB-INF/lib/bcprov-jdk18on-1.72.jar" use-physical-code-source="true"/>
+			<resource-root path="WEB-INF/lib/bcprov-jdk18on-1.76.jar" use-physical-code-source="true"/>
```

Comment:
Let's do this quick fix. Later on, I will try to investigate maven resource filtering.